scripts/check-commit-message.js

```diff
@@ -106,6 +106,36 @@ const checkSecoundLine = (line) => {
     return issues;
 };
 
+const checkPackagesModified = (files) => {
+    const issues = [];
+    const packages = files.reduce((pkgs, file) => {
+        const pathParts = file.split('/');
+
+        if (pathParts[0] === 'packages') {
+            pkgs.add(pathParts[1]);
+        }
+
+        return pkgs;
+    }, new Set());
+
+    if (packages.size > 1) {
+        issues.push(`[Line 1] A breaking change cannot affect to more than one package. Packages affected:\n    - ${[...packages].join('\n    - ')}`);
```

Comment:
The `[Line 1]` is used to refer to the first line of the commit message. Given we're not technically validating the message itself I think we should omit that here.

```suggestion
        issues.push(`A breaking change cannot affect to more than one package. Packages affected:\n    - ${[...packages].join('\n    - ')}`);
```